janus_server/src/datastore.rs

```diff
@@ -817,33 +817,33 @@ pub mod test_util {
     use deadpool_postgres::{Manager, Pool};
     use lazy_static::lazy_static;
     use std::str::{self, FromStr};
-    use testcontainers::{clients::Cli, images::postgres::Postgres, Container, Docker};
+    use testcontainers::{clients::Cli, images::postgres::Postgres, Container};
     use tokio_postgres::{Config, NoTls};
 
     const SCHEMA: &str = include_str!("../../db/schema.sql");
 
     // TODO(brandon): use podman instead of docker for container management once testcontainers supports it
     lazy_static! {
-        static ref DOCKER: Cli = Cli::default();
+        static ref CONTAINER_CLIENT: Cli = Cli::default();
     }
 
     /// DbHandle represents a handle to a running (ephemeral) database. Dropping this value causes
     /// the database to be shut down & cleaned up.
-    pub struct DbHandle(Container<'static, Cli, Postgres>);
+    pub struct DbHandle(Container<'static, Postgres>);
 
     /// ephemeral_datastore creates a new Datastore instance backed by an ephemeral database which
     /// has the Janus schema applied but is otherwise empty.
     ///
     /// Dropping the second return value causes the database to be shut down & cleaned up.
     pub async fn ephemeral_datastore() -> (Datastore, DbHandle) {
         // Start an instance of Postgres running in a container.
-        let db_container = DOCKER.run(Postgres::default().with_version(14));
+        let db_container = CONTAINER_CLIENT.run(Postgres::default());
```

Comment:
We should add `.with_tag("14-alpine")` to pin our Postgres version here.
```suggestion
        let db_container = CONTAINER_CLIENT.run(RunnableImage::from(Postgres::default()).with_tag("14-alpine"));
```